xbmc/guilib/GUIControlFactory.cpp

```diff
@@ -963,8 +961,18 @@ CGUIControl* CGUIControlFactory::Create(int parentID, const CRect &rect, TiXmlEl
   {
     StringUtils::ToLower(strTmp);
     if (strTmp == "horizontal")
+    {
       orientation = HORIZONTAL;
+      // default orientation for a settings slider is vertical
```

Comment:
it's common practice to break skins during development cycles. we do it all the time.